ui/src/dashboards/components/CellEditorOverlay.js

```diff
@@ -179,6 +183,7 @@ class CellEditorOverlay extends Component {
               onSelectGraphType={this.handleSelectGraphType}
               onCancel={onCancel}
               onSave={this.handleSaveCell}
+              isSavable={queriesWorkingDraft.every(isQuerySavable)}
```

Comment:
I just clarified in a new review the reasoning behind my comment here. I don't believe I'm nit-picking but rather pointing out how logic needs to be changed in order for this to work as intended, but please let me know if I'm incorrect.

As for naming with `isNot`, that's fine-- a `!` would work just as well.

And good point on on the `FullyQualified` semantics.